app/models/spree/vendor_ability.rb

```diff
@@ -79,7 +79,7 @@ def apply_product_properties_permissions
   end
 
   def apply_prototypes_permissions
-    can :read, Spree::Prototype
+    can [:read, :admin], Spree::Prototype
```

Comment:
```suggestion
    can [:read, :display], Spree::Prototype
```

maybe this will work? giving them admin privilages is very risku

Reply:
Unfortunately `:display` is not working. I also tried `:show` and `:index` but without success.

Reply:
However, I noticed that `:admin` action does not give the ability to edit or delete prototypes. Vendor user can only see a list of prototypes:
<img width="1387" alt="Screenshot 2020-07-09 17 31 47" src="https://user-images.githubusercontent.com/5177038/87060198-87154100-c20a-11ea-823d-4f8251bcc8bd.png">
So maybe it is not as bad, safety-wise.